west.yml

```diff
@@ -16,22 +16,18 @@ manifest:
 
   projects:
     - name: zephyr
-      remote: retfie
-      revision: helium_mapper.mcuboot
+      remote: upstream
```

Comment:
This requires `west update -f always` to fetch zephyr and mcuboot modules to their current main.
Please add a small note about this into README.rst.